xbmc/settings/AdvancedSettings.cpp

```diff
@@ -914,8 +914,8 @@ void CAdvancedSettings::ParseSettingsFile(const std::string &file)
     XMLUtils::GetInt(pElement, "mincommbreaklength", m_iEdlMinCommBreakLength, 0, 5 * 60);  // Between 0 and 5 minutes
     XMLUtils::GetInt(pElement, "maxcommbreakgap", m_iEdlMaxCommBreakGap, 0, 5 * 60);        // Between 0 and 5 minutes.
     XMLUtils::GetInt(pElement, "maxstartgap", m_iEdlMaxStartGap, 0, 10 * 60);               // Between 0 and 10 minutes
-    XMLUtils::GetInt(pElement, "commbreakautowait", m_iEdlCommBreakAutowait, 0, 10);        // Between 0 and 10 seconds
-    XMLUtils::GetInt(pElement, "commbreakautowind", m_iEdlCommBreakAutowind, 0, 10);        // Between 0 and 10 seconds
+    XMLUtils::GetInt(pElement, "commbreakautowait", m_iEdlCommBreakAutowait, 0, 60);        // Between 0 and 60 seconds
```

Comment:
changing to
```suggestion
    XMLUtils::GetInt(pElement, "commbreakautowait", m_iEdlCommBreakAutowait, -60, 60);        // Between -60 and 60 seconds
```
allows me to enter a value of -2 here in advancedsettings.xml
This allows me to trim off more of the commercial break so I only have to look at the pre and post commercial roll, rather than any of the commercials themselves.
Suggest allowing +/- 60 seconds for autowait and autowind will allow individual users to tweak the edl behaviour to their liking based on how well their particular back end’s commflagging method aligns with the add break start and end, and for personal taste regarding how much add they want to see to make sure they didn’t miss any programme.